synapse/handlers/presence.py

```diff
@@ -329,7 +330,7 @@ def __init__(self, hs):
 
         # user_id -> last_sync_ms. Lists the users that have stopped syncing but
         # we haven't notified the presence writer of that yet
-        self.users_going_offline = {}
+        self.users_going_offline = {}  # type: Dict[str, int]
```

Comment:
Is there a reason not to write the type as `self.users_going_offline: Dict[str, int] = {}` other than consistency?

Reply:
Nope, I keep forgetting that is a thing we are allowed to do now